clients/go/cmd/zbctl/testdata/deploy_resources.golden

```diff
@@ -41,6 +41,14 @@
         "resourceName": "drg-force-user.dmn",
         "tenantId": "<default>"
       }
+    },
+    {
+      "form": {
+        "formId": "form",
+        "version": 1,
+        "formKey": "2251799813685251",
+        "resourceName": "form.form"
```

Comment:
❌ It also fails on this like, as the resource name is `test-form-1.form`

Reply:
Updated! 👍